app/controllers/api_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ApiController < ApplicationController
+  def service_interaction_for_local_authority
+    @authority = LocalAuthority.find_by_slug!(params[:authority_slug])
+    @service = Service.find_by(lgsl_code: params[:lgsl])
+    interactions = @service.interactions
+    @link = links.detect { |l| interactions.first.service_interactions.first.id == l.service_interaction.id }
+
+    render json: local_interaction_response
+  end
+
+private
+
+  def local_interaction_response
+    {
+      "local_authority"=>{
+        "name"=>@authority.name,
```

Comment:
Frontend needs to know whether the URL is for an interaction or the homepage, because we use [different text](https://github.com/alphagov/frontend/blob/3b12a066f0787f9a926912becf04a8b440792f12/config/locales/en.yml#L38) on the page, eg https://www.gov.uk/pay-council-tax/dundee